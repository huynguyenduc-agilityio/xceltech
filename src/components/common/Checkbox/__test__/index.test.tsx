import { render } from '@testing-library/react';

import { Checkbox } from '..';

describe('Avatar component', () => {
  it('should match snapshot', () => {
    const { container } = render(<Checkbox />);

    expect(container).toMatchSnapshot();
  });
});
