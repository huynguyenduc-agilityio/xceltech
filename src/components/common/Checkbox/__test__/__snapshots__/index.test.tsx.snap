// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component should match snapshot 1`] = `
<div>
  <button
    aria-checked="false"
    class="peer w-[30px] h-[26px] shrink-0 rounded-md border-2 border-gray-muted ring-offset-background focus-visible:outline-none disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-transparent data-[state=checked]:text-primary"
    data-state="unchecked"
    role="checkbox"
    type="button"
    value="on"
  />
</div>
`;
